_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
I was thinking something similar!  I'd like to put a page together with a call to action to encourage folks to get involved.